### ISSUE TITLE
Compiling project source code from Windows 10 x64

### ISSUE THREAD

**Issue by svanimisetti** [hoiberg/bluetoothMidi#1]
@hoiberg - Thanks for sharing your Steinway project! I really enjoyed reading it and congratulate you on a great execution! I am trying to follow your project but have few roadblocks. This is technically not an issue - but documenting it here.

Initially, I started with the ESP32 - but then realized I had a Pro Micro and HM-10 lying in the parts bin. I stumbled across your project recently. I tried to follow your project along - but soon realized the need for this "patch" app to bridge the MIDI payload between Bluetooth serial from HM10 to iOS CoreMIDI. I understand that this limitation is due to HM-10 not being able to emulate the 128-bit UUID required of a Bluetooth MIDI device.

Unfortunately, I am a Windows 10 x64 person - and do not have macOS at my disposal to compile the app. I see that this app is no longer available on the iTunes app store. I tried other apps (e.g. midimittr by Matt Sieren) - but they require a Bluetooth MIDI class device as well. I also do not have an Apple developer account. I am not sure if I can build the app using your GitHub repo on a Windows machine and upload it into my iPhone/iPad (just for personal installation). I am not sure if Apple allow apps to be developed this way - even if I do not intend to distribute them.

If you have any suggestions / recommendations, please advise.

**Comment by hoiberg** [comment]
As far as I know, it is not possible to compile and deploy XCode projects on a Windows machine. There may be services that offer remote macOS access for a small fee (but I don't know how you would install it then on your device), or maybe you know someone with a Mac who could lend you his/her machine.

You could use a cross-platform solution like Xamarin, but that would require you to re-write the whole app.

What you also have to keep in mind is that I have not updated this project in years, so it might require some work before it compiles again.

**Comment by svanimisetti** [comment]
Thanks for the insight. In the meantime, I changed my approach a little. I think bluetooth to be a difficult wireless medium - it does not seem to work seamlessly between all of my devices (Win10 PC, Win10 laptop, iPad, iPhone). I found a "seamless" solution with ESP8266 using [AppleMIDI ](https://github.com/lathoub/Arduino-AppleMIDI-Library) library over wifi. I already tested this with my Win10 PC and iPad - and I am able to use the same configuration across the two devices - not patch work of apps/devices! On Win10, I use Tobias Erichsen's [rtpMIDI](https://www.tobias-erichsen.de/software/rtpmidi.html) and on iOS it is Oscar Bouwman's [MIDI network](https://apps.apple.com/us/app/midi-network/id1102428729). With this arrangement - I am able to work with multiple apps across all devices (primarily, Synthesia and MuseScore).

Of course, there are latencies if the wifi network is crowded, but I am able to configure ESP8266 in soft access point mode - and the performance is much better than bluetooth. I am hoping to make good progress on my project in the coming months. Nontheless, thanks for your time! I appreciate your sharing your work.
